tests/pathfinding/test_capacity_updates.py

```diff
@@ -58,27 +56,31 @@ def get_updatepfs_message(
         updating_capacity=updating_capacity,
         other_capacity=other_capacity,
         reveal_timeout=reveal_timeout,
-        signature=signature,
     )
 
+    updatepfs_message.sign(LocalSigner(privkey_signer))
+
+    return updatepfs_message
+
 
 def test_pfs_rejects_capacity_update_with_wrong_chain_id(
-    addresses: List[Address], pathfinding_service_mocked_listeners: PathfindingService
+    pathfinding_service_mocked_listeners: PathfindingService,
 ):
 
     message = get_updatepfs_message(
         chain_identifier=ChainID(121212),
-        updating_participant=addresses[0],
-        other_participant=addresses[1],
+        updating_participant=private_key_to_address(PRIVAT_KEY_EXAMPLE_1),
+        other_participant=private_key_to_address(PRIVAT_KEY_EXAMPLE_2),
+        privkey_signer=PRIVAT_KEY_EXAMPLE_1,
```

Comment:
`PRIVAT_KEY_EXAMPLE_1` is the default, so you don't strictly need this here. But it might be nice for readability. (same is true some more times below.)

Reply:
you are right. Let's keep it for readability? I strongly suggest that :)